#example_name:
#- [x, y, z]
#- [roll, pitch, yaw]

area_1:
- [0.5, -0.3, 0.0]
- [3.14, 0.0, 0.0]

area_2:
- [0.5, 0.0, 0.0]
- [3.14, 0.0, 0.0]

area_3:
- [0.5, 0.3, 0.0]
- [3.14, 0.0, 0.0]

area_4:
- [0.2, 0.4, 0.0]
- [3.14, 0.0, -0.78]

area_5: # twisted
- [0.2, -0.4, 0.0]
- [3.14, 0.0, 0.78]
